core/iwasm/interpreter/wasm_loader.c

```diff
@@ -250,36 +250,72 @@ loader_malloc(uint64 size, char *error_buf, uint32 error_buf_size)
 static bool
 check_utf8_str(const uint8* str, uint32 len)
 {
-    const uint8 *p = str, *p_end = str + len, *p_end1;
-    uint8 chr, n_bytes;
+    // The valid ranges are taken from page 125, below link
```

Comment:
Could we use /* */ to comment? // is C++ style, warning is reported by some compilation.